apps/web/src/components/NavMenu/IAPNavMenu.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { MenuLink } from "@gc-digital-talent/ui";
+import { Maybe, User } from "@gc-digital-talent/graphql";
+
+import useRoutes from "~/hooks/useRoutes";
+
+import NavMenu from "./NavMenu";
+import LogoutConfirmation from "../LogoutConfirmation";
+import { LogoutButton } from "../Layout/Layout";
+
+interface IAPNavMenuProps {
+  loggedIn?: boolean;
+  user?: Maybe<User>;
+}
+
+const IAPNavMenu = ({ loggedIn, user }: IAPNavMenuProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+
+  let authLinks = [
+    <MenuLink key="login-info" to={`${paths.login()}?iap`}>
+      {intl.formatMessage({
+        defaultMessage: "Login",
+        id: "md7Klw",
+        description: "Label displayed on the login link menu item.",
+      })}
+    </MenuLink>,
+    <MenuLink key="register" to={`${paths.register()}?iap`}>
+      {intl.formatMessage({
+        defaultMessage: "Register",
+        id: "LMGaDQ",
+        description: "Label displayed on the register link menu item.",
+      })}
+    </MenuLink>,
+  ];
+
+  if (loggedIn && user) {
+    authLinks = [
+      <LogoutConfirmation key="logout">
+        <LogoutButton>
+          {intl.formatMessage({
+            defaultMessage: "Logout",
+            id: "3vDhoc",
+            description: "Label displayed on the logout link menu item.",
+          })}
+        </LogoutButton>
+      </LogoutConfirmation>,
+    ];
+  }
+
+  return (
+    <NavMenu
+      utilityItems={authLinks}
+      mainItems={[
+        <MenuLink key="home" to={paths.iap()}>
+          {intl.formatMessage({
+            defaultMessage: "IT Apprenticeship Program for Indigenous Peoples",
+            id: "k4Vsh0",
+            description:
+              "Link to the homepage for IT Apprenticeship Program for Indigenous Peoples.",
+          })}
+        </MenuLink>,
+        <MenuLink key="home" to={paths.home()} end>
```

Comment:
Watch the keys.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/dec3e8af-1616-4be9-9f1a-cc93e20031e3)

Reply:
Copy paste 🤦‍♀️ Fixed in [update with unique keys](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6993/commits/98a5b711be04056070c78ed22864d75314591b8a)

Reply:
:+1: